ncdomain/convert.go

```diff
@@ -1083,3 +1092,55 @@ func (v *Value) moveEmptyMapItems() {
 		}
 	}
 }
+
+// makeJSONSafe walks an interface to ensure all maps use string keys so that
```

Comment:
Can you include an attribution to exactly where this code came from? The GitHub issue link doesn't make it clear.